### DISCUSSION TITLE
Problème pour activer l'envoi de mail

### DISCUSSION THREAD

**Discussion by moiiom59** [dbarzin/mercator#506] [General]
Bonjour,
Je n'arrive pas à parametrer l'envoi de mail . J'avoue ne pas bien comprendre le principe . Le service smtp et local ? je dispose d'un serveur de messagerie exchange qui pourrait être relai.
merci de votre aide

**Comment by dbarzin** [comment]
Tu dois te faire assister par le sysadmin de ton organisation. Mettre en place l'envoi de ail est généralement bien décrit dans la documentation de la distribution Linux que tu utilises.
